### PR TITLE
updates title

### PR DESCRIPTION
**Pull request by HassanHassouna**

### CODE REVIEW

**Comment by hhasona** [comment]
@karyum

**Comment by HassanHassouna** [comment]
@amirbardugo

File: src/ex1/index.html [hunk 1]
```diff
@@ -1,14 +1,39 @@
...
-
-</body>
-<script src="script.js"></script>
+  </head>
+  <body>
+    <fieldset class="container__field">
+      <div class="container">
+        <h1>Todo App</h1>
+        <form action="">
+          <div class="antd">
```
Comment (amirbardugo) [inline src/ex1/index.html:12]: what is "antd'? try to give more explanatory class names :)

File: src/ex1/index.html [hunk 2]
```diff
@@ -1,14 +1,39 @@
...
+            <input
+              class="ant-input"
+              id="input"
+              type="text"
+              name="todo"
+              placeholder="Add your new todo"
+            />
+            <input id="addButton" class="addbtn" type="button" value="+" />
```
Comment (amirbardugo) [inline src/ex1/index.html:20]: its better to use <button> for buttons, this way you'll have more control on the button, you will be able to put icons/images or whatever you'll want later on
also IMO its more readable and clear that way

File: src/ex1/index.html [hunk 3]
```diff
@@ -1,14 +1,39 @@
...
+          </div>
+        </form>
+        <ul id="todolist" class="todolist">
+          <!-- adding todo lists -->
+        </ul>
+        <div id="error"></div>
```
Comment (amirbardugo) [inline src/ex1/index.html:26]: just a syntax suger, you can use self closing element
```suggestion
        <div id="error" />
```

File: src/ex1/script.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+function deleteAllTasks() {
+  const list = document.getElementById("todolist")
+  const counter = document.getElementById("counter")
+  counter.innerHTML = 0
+  list.innerHTML = ""
+}
+
+function validationText() {
```
Comment (amirbardugo) [inline src/ex1/script.js:69]: does this function only validates the text? lets give her a more indicative name 🙂

File: src/ex1/script.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  const input = document.getElementById("input")
+  if (input.value === "") {
+    errorMessage.innerHTML = "Please enter a task"
+    errorMessage.style.display = "block"
+    errorMessage.style.transition = "all 0.5s ease-in-out"
+    errorMessage.style.color = "red"
+    errorMessage.style.fontSize = "20px"
+    errorMessage.style.fontWeight = "bold"
+    errorMessage.style.textAlign = "center"
+    errorMessage.style.marginTop = "10px"
```
Comment (amirbardugo) [inline src/ex1/script.js:79]: you can define all of those only one time in your CSS file, and when error appears only change the actual text message and the visibility (e.g- display:block)

File: src/ex1/script.js [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+    countLiDeleted()
+  })
+  li.addEventListener("mouseover", function () {
+    deleteButton.style.display = "block"
+    deleteButton.style.transition = "all 0.5s ease-in-out"
+  })
+  li.addEventListener("mouseout", function () {
+    deleteButton.style.display = "none"
+    deleteButton.style.transition = "all 0.5s ease-in-out"
+  })
```
Comment (amirbardugo) [inline src/ex1/script.js:40]: its a possible solution, but you can use here the CSS :hover selectors, its usually better to use them when possible, if you'r having problem with that contact me 😃

File: src/ex1/script.js [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+  const counter = document.getElementById("counter")
+  counter.innerHTML = count + 1
+}
+
+function countLiDeleted() {
+  const list = document.getElementById("todolist")
+  const count = list.getElementsByTagName("li").length
+  const counter = document.getElementById("counter")
+  counter.innerHTML -= 1
+}
```
Comment (amirbardugo) [inline src/ex1/script.js:60]: those two shares some code, you could change it to one function that gets a parameter or extract the 3 first line to a function named getCurrentCounterValue

File: src/ex1/script.js [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+  const listItems = list.getElementsByTagName("li")
+  const listArray = Array.from(listItems)
+  listArray.sort(function (a, b) {
+    return a.innerHTML.toLowerCase().localeCompare(b.innerHTML.toLowerCase())
+  })
+  list.innerHTML = ""
+  listArray.forEach(function (item) {
+    list.appendChild(item)
+  })
+}
```
Comment (amirbardugo) [inline src/ex1/script.js:98]: cool! 🔥

**Review by amirbardugo — COMMENTED** [review]
Overall great job!
take a look on the comments and let me know if you have any questions or something is unclear
you forgot the alert part, did you encounter any issues with that?

**Pull request merged**